File: SimplePigLatin.py
def pig_it(s):
    words = s.split()
    transformed = []

    for word in words:
        if len(word) > 1:
            transformed.append(word[1:] + word[0] + "ay")
        elif len(word) == 1 and word.isalpha():
            transformed.append(word + "ay")
        else:
            transformed.append(word)
    
    return " ".join(transformed)

File: Javascript/SimplePigLatin.js
const pigIt = (a) =>  {
    return a.split(" ").map(elt => {
      if(elt.length > 1) {
        return elt.slice(1) + elt[0] + "ay";
      } 
      else if (elt.length === 1 && 
        ((elt.charCodeAt(0) >= 65 && elt.charCodeAt(0) <= 90) || 
        (elt.charCodeAt(0) >= 97 && elt.charCodeAt(0) <= 122))) {
        return elt + "ay";
      }
      return elt;
    }).join(" ");
  }